src/repos/UserRepo.ts

```diff
@@ -24,11 +25,12 @@ const createUser = async (
       return Object.assign(new PublicationID(), { id });
     });
     const uuid = uuidv4();
-
+    const creationDate = new Date();
+    const expirationDate = new Date(IMMEDIATELY_EXPIRE_DATE);
     const weeklyDebrief = Object.assign(new WeeklyDebrief(), {
       uuid,
-      creationDate: new Date(),
-      expirationDate: new Date('December 17, 1995 03:24:00'),
+      creationDate,
+      expirationDate,
```

Comment:
I'm not too familiar with the codebase, but what is the point of making this "invalid" `WeeklyDebrief` object? Can you not define the User model's `weeklyDebrief` field as `weeklyDebrief: WeeklyDebrief | undefined;`, for example? Sounds like a super jank workaround that could confuse future members working on this...

Reply:
I see what IMMEDIATELY_EXPIRE_DATE is for now; ideally we can leave `weeklyDebrief` as `undefined` as @snajima mentioned until the weekly debriefs are generated for the first time in which case we can have a valid creation and expiration date.